models.py

```diff
@@ -12,12 +14,18 @@
 )
 
 
+class Complexity(enum.Enum):
+    _easy = "easy"
+    _medium = "medium"
+    hard = "hard"
```

Comment:
why the format is different? _easy _medium hard

Reply:
We need to sort exercises by complexity, and it's good to have complexity as strings, not as numbers. When sorting strings as-is the order becomes "easy, hard, medium", I tried using a custom order function, but it didn't work

This hack is a bit ugly but works perfectly for ordering :D